src/v2i-hub/PortDrayagePlugin/src/PortDrayagePlugin.cpp

```diff
@@ -78,30 +111,90 @@ void PortDrayagePlugin::HandleMobilityOperationMessage(tsm3Message &msg, routeab
 
 	// Retrieve J2735 Message
 	auto mobilityOperation = msg.get_j2735_data();
-	FILE_LOG(logERROR) << "Body OperationParams : " << mobilityOperation->body.operationParams.buf;
-	FILE_LOG(logERROR) << "Body Strategy : " << mobilityOperation->body.strategy.buf;
-
+	
+	// String Stream for strategy and operationParams
 	std::stringstream strat;
 	std::stringstream payload; 
  
+	// Create ptree object for json
 	ptree pr;
+
+	// Create new PortDrayage_Object pointer
 	PortDrayage_Object *pd = new PortDrayage_Object();
+
+	// Read strategy and operationParams
 	strat << mobilityOperation->body.strategy.buf;
 	payload << mobilityOperation->body.operationParams.buf;
 
+	// Compare strategy to PortDrayage strategy
 	std::string strategy = strat.str();
 	if ( strategy.compare(PORT_DRAYAGE_STRATEGY) == 0 ){
 		try {
+			PLOG(logERROR) << "Body OperationParams : " << mobilityOperation->body.operationParams.buf;
+			PLOG(logINFO) << "Body Strategy : " << mobilityOperation->body.strategy.buf;
 			// Convert JSON payload to PortDrayage_Object
 			read_json(payload, pr);
 			*pd = readPortDrayageJson( pr );
+
+			// Handle actions that require PortDrayage WebService Input
+			PLOG(logDEBUG) << "Operation is " << pd->operation << std::endl;
+			if ( pd->operation.compare("PICKUP") == 0 ) {
```

Comment:
maybe use case insensitive comparison, and trim spaces

Reply:
Not necessary. This enum should never have different capitalization or spaces